bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -163,13 +164,14 @@ public void getURLInput(String url) throws IOException {
                     updateState(CURRENT_TOTAL, new DecimalType(myObject.get("total").toString()));
                     String timestamp = myObject.get("startsAt").toString().substring(1, 20);
                     updateState(CURRENT_STARTSAT, new DateTimeType(timestamp));
+                    updateState(CURRENT_LEVEL, new StringType(myObject.get("level").toString()));
 
                 } catch (JsonSyntaxException e) {
                     updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                             "Error communicating with Tibber API: " + e.getMessage());
                 }
             }
-            if (jsonResponse.contains("daily")) {
+            if (jsonResponse.contains("daily") && !jsonResponse.contains("[]") && !jsonResponse.contains("\"daily\":null")) {
```

Comment:
I feel that `!jsonResponse.contains("[]")` might be too broad of a check with regards to checking the entire response. You should make sure to narrow down the check to just the `daily` field.

Reply:
Good point, narrowed down to `"daily":{"nodes":[]`